### PR TITLE
Publishing workflow

### PR DESCRIPTION
**Pull request by Edoardo-Pedicillo**
In this PR, @MatteoRobbiati @AleCandido and I have deployed the workflow to publish the`latest`and `stable` Qibocal's documentation, and we have made minor fixes to `conf.py` and `pyproject.toml`.

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.

### CODE REVIEW

**Review by andrea-pasquale — APPROVED** [review]

**Comment by andrea-pasquale** [comment]
As @scarrazza suggested in qiboteam/qibo#742 we need to fix pre-commit.
I think that there are a few issues given that we are trying to merge a branch from a different organization which is private.
My suggestion is to run pre-commit locally on all files using:

```
pre-commit run --all-files
```
then commit and push.

**Comment by Edoardo-Pedicillo** [comment]
Thanks @andrea-pasquale, it works.

File: doc/source/conf.py [hunk 1]
```diff
@@ -17,16 +17,15 @@
 
 sys.path.insert(0, os.path.abspath(".."))
 
-import qibocal
 
 # -- Project information -----------------------------------------------------
 
-project = "qibocal"
+project = " "
```
Comment (scarrazza) [inline doc/source/conf.py:23]: Please follow the suggestion proposed in: https://github.com/qiboteam/qibo/pull/745

**Comment by AleCandido** [comment]
> merge a branch from a different organization which is private.

I agree there might be extra issues, but there are no private orgs, and the fork is also public.

**Comment by scarrazza** [comment]
I would like to merge this asap, however the `conf.py` seems not really following the final decision discussed in https://github.com/qiboteam/qibo/pull/745, could you please have a look?

**Comment by MatteoRobbiati** [comment]
> I would like to merge this asap, however the `conf.py` seems not really following the final decision discussed in [qiboteam/qibo#745](https://github.com/qiboteam/qibo/pull/745), could you please have a look?

Thank you. I think this is now aligned with the `qiboteam`'s one.

**Pull request merged**